mscore/musescore.cpp

```diff
@@ -8038,11 +8067,12 @@ void MuseScore::init(QStringList& argv)
             TourHandler::addWidgetToTour("welcome", menubar, "menubar");
 
       if (!scoresOnCommandline && preferences.getBool(PREF_UI_APP_STARTUP_SHOWSTARTCENTER) && (!restoredSession || mscore->scores().size() == 0)) {
+            showSplashMessage(sc, tr("Initializing start center…"));
```

Comment:
“Start Center” should be capitalized, as seen in the [Handbook](https://musescore.org/en/handbook/create-new-score#start-center).

Reply:
;-) This will not be changed as discussed on Telegram.

Reply:
We didn't resolve this either way.

Reply:
Really?